src/main/docs/guide/httpServer/serverConfiguration/cors/corsAllowedOrigins.adoc

```diff
@@ -18,4 +18,4 @@ micronaut:
             - http://foo.com
 ----
 
-WARNING: Use the `allowed-origins-regex` configuration judiciously. You may accidentally make an insecure configuration which could be targeted by an attacker registering domains targeting the regular expression.
+WARNING: Use the `allowed-origins-regex` configuration judiciously. You may accidentally make an insecure configuration which could be targeted by an attacker registering domains targeting the regular expression. Note that if `allowed-origins-regex` is set then `allowed-origins` is ignored.
```

Comment:
is that so when you use cors via configuration?

Reply:
I don't think configuration prevents setting both. I'm checking to see what happens. I'll write a test.

Reply:
OK, it looks like with configuration, you use something like

```
micronaut.server.cors.configurations.foo.allowed-origins-regex = ^http(|s):\\/\\/foo\\.com$
micronaut.server.cors.configurations.foo.allowed-origins = https://bar.com
```

then both `http://foo.com` and `https://bar.com` are allowed.

I stand corrected and will change annotation-based to match.